elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -327,7 +327,7 @@ private void bindAttrOrRelation(AccessibleObject fieldOrMethod) {
         String fieldName = getFieldName(fieldOrMethod);
         Class<?> fieldType = getFieldType(entityClass, fieldOrMethod);
 
-        if (fieldName == null || "id".equals(fieldName) || "class".equals(fieldName)
+        if (fieldName == null || "class".equals(fieldName)
```

Comment:
Since we can have ID fields that are not "id", this implies that some customer could have an ID field called "jobId" and another non-ID field called "id". For that reason, I removed "id" from reserved field lisst

Reply:
"id" is reserved in JSON-API.   From the JSON-API spec:
> In other words, a resource can not have an attribute and relationship with the same name, nor can it have an attribute or relationship named type or id.

Based on this, the original code is correct.

Reply:
Restored